kodein/build.gradle

```diff
@@ -37,58 +21,13 @@ dependencies {
 
     testCompile "junit:junit:$junit_version"
 
-    testCompile project(':kodein-core').sourceSets.test.output
```

Comment:
@SalomonBrys For some reason I couldn't get this to work. Does it need to be here?